PACKAGING.md

```diff
@@ -0,0 +1,58 @@
+How we package ka-lite
+======================
+
+*March 20th 2015*
+
+Introduction
+------------
+
+Our project, ka-lite, was not always intended to be a python package suitable
+for PyPi and the likes. However, time has changed that, and we are currently
+in the process of making it possible to distribute through traditional
+and conventional methods.
+
+Currently, the whole `kalite` package is built for being run on its own and
+integrates badly with outside Django environments, thus we also package it as
```

Comment:
It integrates badly, but that's not *why* we bundle it -- we bundle it because we're often installing offline in contexts where these external packages aren't available... Might be good to clarify the intent here.